SalesforceCommerceCloud/pwa-kit
# [V2] Fix localization scripts to output to the correct default locale

# Description

This PR makes the localization-related npm scripts to behave the same way as in V1. We had hardcoded the default locale `en-US` for the Retail React App in V1, but had not done so for V2.

Otherwise, running `npm run build` or `npm run extract-default-translations` would result in generating `translations/.json` file (no file name). Why? Because we no longer have a key in the package.json that is referred to by `$npm_package_l10n_defaultLocale`.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Hardcoding the `en-US` default locale for the Retail React App

# How to Test-Drive This PR

1. `cd packages/template-retail-react-app`
2. `npm run extract-default-translations`
3. Check your git status and verify that there is no new translation file generated → the output should be identical to the existing `translations/en-US.json`

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
